pkg/config/config_template.yaml

```
@@ -1386,6 +1386,38 @@ api_key:
     #
     # enabled: false
 
+  ## @param memory_controller - custom object - optional
+  ## Cgroup memory controller for internal memory profiling.
+  ##
+  ## memory_controller:
+  #
+    ## @param enabled - boolean - optional - default: false
+    ## Enable cgroup memory controller.
+    #
+    # enabled: false
+    #
+    ## @param thresholds - map of strings - optional
+    ## Thresholds and the respective active actions to trigger when
+    ## memory usage is above the specified threshold.
+    ## Threshold can be either an absolute value - such as 500MB or 2GB -
+    ## or a percentage of the cgroup allocated memory such as 50%.
+    ## The action can be either:
```

Either implies one or the other (not three options).

```suggestion
    ## The action can be:
```